assets/src/domain/eventEditor/ui/tickets/ticketForm/multiStep/Content.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useCallback } from 'react';
+import { __ } from '@wordpress/i18n';
+
+import { EspressoForm } from '@application/ui/forms/espressoForm';
+import useTicketFormConfig from '../useTicketFormConfig';
+import ContentWrapper from './ContentWrapper';
+import { useTicketMutator } from '@edtrServices/apollo/mutations';
+import { ContentProps } from './types';
+import { useMutatePrices } from '@edtrUI/tickets/ticketPriceCalculator/hooks';
+
+const Content: React.FC<ContentProps> = ({ entity, onClose }) => {
+	const { createEntity, updateEntity } = useTicketMutator();
+	const mutatePrices = useMutatePrices();
+
+	const onSubmit = useCallback(
+		({ prices, deletedPrices, ...fields }) => {
+			mutatePrices(prices, deletedPrices).then((relatedPriceIds) => {
```

Comment:
where is the `catch` ?
or errors are handled in a different part of the app?

Reply:
We have mutation handler to display error notice.